credsweeper/rules/config.yaml

```diff
@@ -424,7 +424,7 @@
   severity: high
   type: pattern
   values:
-    - (?P<value>[0-9]{10}:AA[\w\\-_-]{33})
+    - (?P<value>[0-9]{8,10}:[0-9A-Za-z_-]{35})
```

Comment:
Is `Telegram Bot API Token` has been changed?
If it is changed, this change should be included in other PR.
Please check.

Reply:
There are many samples of the token type, but not any rule.
https://core.telegram.org/bots/api#authorizing-your-bot
https://core.telegram.org/bots/features#creating-a-new-bot
I think, left numeric side is grown and now it is 10 symbols, but oldest tokens might be with 9 or less symbols (as it is shown in examples)
Current pattern id matched extra charachters ``^`` and ``]``  due the seuqence ``\\-_`` 
I propose to fix it in the PR:
``- (?P<value>[0-9]{10}:AA[\w-]{33})``

Reply:
Okay.
Let's fix the pattern.